massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -681,6 +682,7 @@ impl ConsensusWorker {
 
                 // Store block in shared storage.
                 self.block_db.storage.store_block(block);
+                self.block_db.storage.extend(storage);
```

Comment:
The block should be in the storage also and so we avoid passing it in the event we just pass the block id ?

Reply:
can you recheck this with the latest conflict resolution ?